package.json

```diff
@@ -45,9 +45,10 @@
     "body-parser": "^1.20.2",
     "class-transformer": "^0.5.1",
     "class-validator": "^0.14.0",
-    "cloudinary": "^1.41.0",
+    "cloudinary": "^1.41.3",
     "date-fns": "^2.30.0",
     "google-auth-library": "^9.0.0",
+    "image-size": "^1.1.1",
     "jwks-rsa": "^3.1.0",
```

Comment:
@CodeGiant1 seems that the package image-size is not needed

Reply:
> @willypelz I just updated my code and pushed it to my branch. Please, feel free to review it now.